### ISSUE TITLE
sys:1: RuntimeWarning: coroutine 'Page.xpath' was never awaited

### ISSUE THREAD

**Issue by shelverr** [pyppeteer/pyppeteer#148]
`async def main():
    browser = await pp.launch(headless=False)
    site = await browser.newPage()
    await site.goto('https://www.google.com/')
    time.sleep(3)
    
    # images = site.xpath("""//*[@id="gbw"]/div/div/div[1]/div[2]/a""")
    await site.click(site.xpath("""//*[@id="gbw"]/div/div/div[1]/div[2]/a"""))
asyncio.get_event_loop().run_until_complete(main())`

My code is throwing this error during "await site.click(site.xpath)"
I'm unsure how to fix this, any help?

**Comment by MadTinker** [comment]
Should post this on stackoverflow not github.

**Comment by Mattwmaster58** [comment]
> Should post this on stackoverflow not github.

Usually yes, because this error is fairly trivial and thus usually appears in the user's code not the library. However, in this case, that's not the case. It's not anything to worry about too much (it's a warning, not an error after all) and it should be fixed in the next release @rshelver

**Comment by shelverr** [comment]
> > Should post this on stackoverflow not github.
> 
> Usually yes, because this error is fairly trivial and thus usually appears in the user's code not the library. However, in this case, that's not the case. It's not anything to worry about too much (it's a warning, not an error after all) and it should be fixed in the next release @rshelver

Okay cool! Do you know of any ways to disregard warnings on pycharm?

**Comment by Mattwmaster58** [comment]
> Okay cool! Do you know of any ways to disregard warnings on pycharm?

No, but there are ways to ignore warnings. I'd recommend you just ignore it for now as warnings can be valuable tools.
